report_xlsx/report/report_xlsx.py

```diff
@@ -18,8 +18,9 @@
 
 class ReportXlsx(report_sxw):
 
+    @api.cr_uid_ids_context
     def create(self, cr, uid, ids, data, context=None):
```

Comment:
Why don't we directly put this `create` in new API format?

Reply:
in that case it fails because of the following call in standard addons:
> /opt/odoo/odoo/oc/odoo/addons/base/ir/ir_actions.py(234)render_report()
-> return report.create(self._cr, self._uid, res_ids, data, context=self._context)

Reply:
Just a question: report_xslx is working fine in v10 as it is now, why do we need to change anything?

Reply:
Old API support was removed in v10, so it's logical to switch this properly for avoiding side effects.

Reply:
I am not sure if Odoo will accept such a PR since technically everything works fine and the focus from Odoo is already on V12.
With this PR this create call can be called from old and new API hence we bypass the issue without touching the code from Odoo. I have done a couple of tests on a number of reports and didn't find any issue when adding the  '@api.cr_uid_ids_context' decorator.
But as stephan says: we can also decide not to change anything and hence close this PR without merge since report_xlsx and report_xlsx_helper work fine since they both use the old API.

Reply:
OK, I let @sbidoul to decide.

Reply:
Since the create method is declared without decorator in the base class, I think it's safer to keep it as is. So I suggest to close this PR.